jsk_fetch_robot/jsk_fetch.rosinstall.kinetic

```diff
@@ -0,0 +1,37 @@
+- git:
+    local-name: jsk-ros-pkg/jsk_recognition
+    uri: https://github.com/jsk-ros-pkg/jsk_recognition
+    version: 1.2.2
```

Comment:
latest jsk_recognitoin is 1.2.5, any trouble?
http://repositories.ros.org/status_page/ros_kinetic_default.html?q=jsk_recognition